downstream/assemblies/builder/assembly-publishing-exec-env.adoc

```diff
@@ -2,5 +2,6 @@
 
 = Publishing an {ExecEnvNameSing}
 
-include::builder/proc-push-execenv-to-hub.adoc[leveloffset=+1]
-include::builder/proc-pull-protected-registry.adoc[leveloffset=+1]
+include::builder/con-gathering-system-dependencies.adoc[leveloffset=+1]
```

Comment:
I thinking publishing ee will only contain information about push and pull, not `con-gathering-system-dependencies.adoc` and `con-pip-based-req.adoc`.

Reply:
push and pull are currently covered in another section "Modifying existing execution environments provided by AAP". Should they be duplicated here?

Reply:
This may be fixed by the Google doc draft changes requested by @nitzmahone.